### PR TITLE
Drop Debian 8 support

### PR DESCRIPTION
**Pull request by bastelfreak**
It's currently in extended support time and even that ends in about a
month.

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -121,8 +121,7 @@
           'passenger_package_name'  => 'libnginx-mod-http-passenger',
           'include_modules_enabled' => true,
         }
-      } elsif ($facts['os']['name'] == 'ubuntu' and $facts['os']['distro']['codename'] in ['lucid', 'precise', 'trusty', 'xenial'])
-      or ($facts['os']['name'] == 'debian' and $facts['os']['release']['major'] in ['6', '7', '8']) {
+      } elsif ($facts['os']['name'] == 'ubuntu' and $facts['os']['distro']['codename'] in ['lucid', 'precise', 'trusty', 'xenial']) {
```
Comment (ekohl) [inline manifests/params.pp:124]: Isn't lucid AFAIK only Xenial is supported here. However, I'd argue we should just drop this and never manage the repo by default. Just trust the OS versions and leave it up to users.

Comment (bastelfreak) [reply manifests/params.pp:124]: I had the same in mind, but I guess it should be handled in a different PR

File: Gemfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem 'voxpupuli-test', '>= 1.0.0',  :require => false
+  gem 'voxpupuli-test', '>= 1.4.0',  :require => false
```
Comment (ekohl) [inline Gemfile:14]: This is modulesynced.

File: manifests/params.pp (hunk 1 above)
Comment (ekohl) [reply manifests/params.pp:124]: Perhaps that should be done first to keep this PR smaller (and the other really focused on just dropping that).

File: Gemfile (hunk 2 above)
Comment (bastelfreak) [reply Gemfile:14]: I know, I wanted a hotfix to get master green again. I will have a look at modulesync_config later :)

**Pull request merged**